src/js/components/Form/stories/Controlled.js

```diff
@@ -17,17 +17,32 @@ import {
 } from 'grommet';
 import { grommet } from 'grommet/themes';
 
+const defaultValue = {
+  name: '',
+  email: '',
+  subscribe: false,
+  ampm: '',
+  size: '',
+  comments: '',
+  age: '',
+};
```

Comment:
This is great!  The documentation for the `Form value` prop was too brief for me to understand it.  This story will help, though I would love to see this level of detail also described in the docs.  It would save a lot of digging and experimentation.